internal/mount/mount_net_use.go

```diff
@@ -43,13 +43,21 @@ func netUse(ctx context.Context, args ...string) (string, error) {
 	out, err := nu.Output()
 	log(ctx).Debugf("net use finished with %v %v", string(out), err)
 
-	return string(out), errors.Wrap(err, "error running 'net use'")
+	errorMsg := "error running 'net use'"
+
+	if err != nil {
+		if errorCode := strings.Replace(err.Error(), "exit status ", "", 1); errorCode == "2" {
+			errorMsg = "this error is often caused by but not limited to the 'WebClient' service being unable to start"
+		}
+	}
+
+	return string(out), errors.WithMessagef(err, errorMsg)
 }
 
 func netUseMount(ctx context.Context, driveLetter, webdavURL string) (string, error) {
 	out, err := netUse(ctx, driveLetter, webdavURL)
 	if err != nil {
-		return "", errors.Wrapf(err, "unable to run 'net use' (%v)", out)
```

Comment:
how about we just add to this string:

"unable to run 'net use" (%v), see https://kopia.io/docs/mounting#windows for more information"

Reply:
Sure, it's much cleaner too. I don't see `/mounting` at the moment. Do you want me to create that page or you do it later yourself?

Reply:
Code changed

Reply:
I don't know why this commit fails the build.

Reply:
it's a random failure. Would you mind adding that page under `site` so that we don't forget it?

Instructions for testing the website are in https://github.com/kopia/kopia/blob/master/BUILD.md

Reply:
Done. Let me know if you want certain part changed.